ci/check_pr.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/sh
+#
+
+# check for any changes in the runtime/ . if
+# there are any changes found, it should mark the PR breaksconsensus and
+# "auto-fail" the PR if there isn't a change in the runtime/src/lib.rs file
+# that alters the version.
+
+# get the commit sha for this PR
+CI_COMMIT_SHA=$(git rev-parse HEAD)
+
+
+boldprint () { printf "|\n| \033[1m${@}\033[0m\n|\n" ; }
+
+boldprint "latest 10 commits of ${CI_COMMIT_REF_NAME}"
+git log --graph --oneline --decorate=short -n 10
```

Comment:
any reason to print this ?

Reply:
This is for debugging purposes, the dev who make the PR can see which commits could have cause the PR to fail, this is the same as the `check_runtime.sh`. The dev also see if this script is checking against the latest commit.